endpoints/errors.go

```diff
@@ -14,41 +14,27 @@ var (
 	// Use NewAPIError() method to create your own.
 
 	// InternalServerError is default error with http.StatusInternalServerError (500)
-	InternalServerError = NewInternalServerError(errorNames[0])
+	InternalServerError = NewInternalServerError("")
 	// BadRequestError is default error with http.StatusBadRequest (400)
-	BadRequestError = NewBadRequestError(errorNames[1])
+	BadRequestError = NewBadRequestError("")
 	// UnauthorizedError is default error with http.StatusUnauthorized (401)
-	UnauthorizedError = NewUnauthorizedError(errorNames[2])
+	UnauthorizedError = NewUnauthorizedError("")
```

Comment:
That was already the case, actually I'm not sure that we should provide these error values.